### PR TITLE
probably working heretics tg port

### PR DESCRIPTION
**Pull request by kiwedespars**
## About The Pull Request
adds pointed spells (no longer will mindswap will require a menu)
ports heretics from tg with a few changes listed down below.

credits to
tgstation/tgstation#51199
tgstation/tgstation#52287
tgstation/tgstation#52385
## Why It's Good For The Game

more antags, more variety.

heretics is a pretty good antag.

## Changelog
:cl:
add: ported heretics.
tweak: mindswap/blind/barnyard curse now all use pointed, which is wonderful if you actually used these spells.

sacrificed players are no longer gibbed, instead husked.
less stamdamage on mansus grasp (80 stam on cit would be yikes)
voiceless dead hp 50 -> 75
revived husks hp 25 -> 40
ash mark mansus grasp reduces cooldown by half, rather than 75%
leeching walk heal per tick 2 -> 3
leeching walk stamina per tick 2 -> 4
leeching walk oxy heal per tick 0.5 -> 1
ascended leeching walk omniheal per tick 3 -> 6
ascended leeching walk stam per tick 10 -> 20
armsy variants now heal 2 hp per tick.
ash man renamed to ashen one.
readded eldritch blade as the starter blade.
lots of smaller tweaks like noise changes.
descriptions touched up
base sickle is now being used again, use it to gain the path versions.
flesh path sickle now has a wound bonus of 5.
ash path sickle now does 3 more damage than the others.
rust path sickle now has 75% embed chance. this is probably overpowered and a mistake.
changed a lot of the evocations.

/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
bearer of the curse
seek
seek
lest

**Comment by Bapldap** [comment]
I contributed B)

**Comment by Ludox235** [comment]
Mansus grasp does barely anything; even without a null rod the person is knocked down but instantly goes back up because you decided to nerf the stamina damage it does into oblivion or something.

Also add a way to make more codexes without an codex, cause if you lose it then uh oh, looks like you just got GAMERED and you're now GBJ'd! Get fucked retard!

**Comment by ghost** [comment]
mansus grasp isn't supposed to be a 1shot or even on its own
stabbing people with your knife buffs your mansus grasp

File: code/modules/antagonists/eldritch_cult/eldritch_items.dm [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+/obj/item/living_heart/attack_self(mob/user)
+	. = ..()
+	if(!IS_HERETIC(user))
+		return
+	if(!target)
+		to_chat(user,"<span class='warning'>No target could be found. Put the living heart on the rune and use the rune to recieve a target.</span>")
+		return
+	var/dist = get_dist(user.loc,target.loc)
+	var/dir = get_dir(user.loc,target.loc)
+
```
Comment (ghost) [inline code/modules/antagonists/eldritch_cult/eldritch_items.dm:19]: needs a check to see if they're on same z-level as you

**Comment by Ludox235** [comment]
> 
> 
> mansus grasp isn't supposed to be a 1shot or even on its own
> stabbing people with your knife buffs your mansus grasp

Yeah no fucking shit it isn't supposed to 1 shot people. But right now it's knocks you down for less than a millisecond, effectively making it completely useless despite the 15 seconds (I THINK SECONDS?) cooldown.

Of course when you upgrade it to abuse the marks of your knife it is way better but yeah it's kinda trash otherwise.

**Comment by kiwedespars** [comment]
it does 40 stamdamage and 15 brute, it's enough to prevent someone getting up for a few seconds. people can still shove themselves up, though, because you can force that 4-5 second timer to get up to be much longer by hitting them once with any weapon.

**Pull request merged**